andela/ah-frontend-prime
# #164798256 Enable users manipulate articles

### What does this PR do?

Enables users to create and read articles

#### Description of Task to be completed?

- Add routes for article creation and viewing
- Create pages that users will interact with to create or read stories

#### How should this be manually tested?

Setup the application as stated [here](https://github.com/andela/ah-frontend-prime/pull/1)

Create and verify  a user - [here](https://github.com/andela/ah-frontend-prime/pull/1)

Create articles with the following routes

- `localhost:8000/create-article` --Simulated [here](ah-frontend-prime-pr-15.herokuapp.com/create-article)

View articles on the landing page and click on them to get the full article --Simulated [here](ah-frontend-prime-pr-15.herokuapp.com)

#### What are the relevant pivotal tracker stories?

[164798256](https://www.pivotaltracker.com/story/show/164798256)

#### Screenshots

Create an article
<img width="1424" alt="Screenshot 2019-05-20 at 09 42 56" src="https://user-images.githubusercontent.com/43245704/58001325-d3bc2b00-7ae3-11e9-8fb9-75fdf6463c52.png">


View all articles
<img width="1424" alt="Screenshot 2019-05-20 at 03 19 46" src="https://user-images.githubusercontent.com/43245704/57990331-40b4ce00-7aae-11e9-8f5f-0c5aa69a897c.png">
